h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -0,0 +1,453 @@
+- **Chi-squared divergence** (``uplift_metric="chi_squared"``) - Euclidean divergence normalized by control group distribution. Asymmetric and also tends to infinity values (if control group distribution contains zero values). :math:`\sqrt{X}(P, Q) = \sum_{i=0}^{N} \frac{\sqrt{p_i-q_i}}{q_i}`
+
+where:
+
+- :math:`P` is treatment group distribution
+- :math:`Q` is control group distribution
+
+In a tree node the result value for split is sum :math:`metric(P, Q) + metric(1-P, 1-Q)`. For the split gain value, the result within the node is normalized using a gini coefficient (Eclidean or ChiSquared) or entropy (KL) for each distribution before and after the split.
+
+You can read more information about ``uplift_metric`` on parameter specification page: `uplift_metric <algo-params/uplift_metric.html>`__.
+
+.. _xrt:
+
+Extremely Randomized Trees
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The same goes for Uplift DRF and for random forests: a random subset of candidate features is used to determine the most discriminative thresholds that are picked as the splitting rule. In extremely randomized trees (XRT), randomness goes one step further in the way that splits are computed. As in random forests, a random subset of candidate features is used, but instead of looking for the most discriminative thresholds, thresholds are drawn at random for each candidate feature, and the best of these randomly generated thresholds is picked as the splitting rule. This usually allows to reduce the variance of the model a bit more, at the expense of a slightly greater increase in bias.
+
+H2O supports extremely randomized trees (XRT) via ``histogram_type="Random"``. When this is specified, the algorithm will sample N-1 points from min...max and use the sorted list of those to find the best split. The cut points are random rather than uniform. For example, to generate 4 bins for some feature ranging from 0-100, 3 random numbers would be generated in this range (13.2, 89.12, 45.0). The sorted list of these random numbers forms the histogram bin boundaries e.g. (0-13.2, 13.2-45.0, 45.0-89.12, 89.12-100).
+
+Defining a Uplift DRF Model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  `model_id <algo-params/model_id.html>`__: (Optional) Specify a custom name for the model to use as
+   a reference. By default, H2O automatically generates a destination
+   key.
+
+-  `training_frame <algo-params/training_frame.html>`__: (Required) Specify the dataset used to build the
+   model. **NOTE**: In Flow, if you click the **Build a model** button from the
+   ``Parse`` cell, the training frame is entered automatically.
+
+-  `validation_frame <algo-params/validation_frame.html>`__: (Optional) Specify the dataset used to evaluate
+   the accuracy of the model.
+
+-  `y <algo-params/y.html>`__: (Required) Specify the column to use as the dependent variable. The data can be numeric or categorical.
```

Comment:
I miss that in the review. There should be only categorical right?

Reply:
Yes, you are right! Thanks!